### PR TITLE
Fix linter warnings

### PR DESCRIPTION
**Pull request by ericgaspar**
- Fix linter warnings

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
     "id": "aeneria",
     "packaging_format": 1,
     "description": {
-        "en": "A little dashboard to analyse your energy consumption data from Linky & weather",
-        "fr": "Un petit tableau de bord pour analyser votre consomation d'énergie à partir des données d'un Linky et de la météo"
+        "en": "Dashboard to analyse your energy consumption data from Linky & weather",
+        "fr": "Tableau de bord pour analyser votre consomation d'énergie à partir des données d'un Linky et de la météo"
```
Comment (SimonMellerin) [inline manifest.json:7]: If you think it's not a little dashboard anymore then thank you :)

File: scripts/install [hunk 2]
```diff
@@ -115,12 +113,8 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 ynh_script_progression --message="Configuring aeneria..." --weight=1
 
 aeneria_conf="$final_path/.env"
-cp ../conf/.env $aeneria_conf
 
-ynh_replace_string --match_string="database_name" --replace_string="${db_name}" --target_file="$aeneria_conf"
-ynh_replace_string --match_string="database_user" --replace_string="${db_user}" --target_file="$aeneria_conf"
-ynh_replace_string --match_string="database_password" --replace_string="${db_pwd}" --target_file="$aeneria_conf"
-ynh_replace_string --match_string="app_secret" --replace_string="${deskey}" --target_file="$aeneria_conf"
+ynh_add_config --template="../conf/.env" --destination="$aeneria_conf"
```
Comment (SimonMellerin) [inline scripts/install:117]: `ynh_add_config` will automatically replace variable ? 

nice !

File: scripts/upgrade [hunk 3]
```diff
@@ -142,12 +145,14 @@ ynh_script_progression --message="Configuring aeneria..." --weight=1
 
 # Copy and set aeneria dist configuration
 aeneria_conf="$final_path/.env"
-cp ../conf/.env $aeneria_conf
+# cp ../conf/.env $aeneria_conf
+
+# ynh_replace_string --match_string="__DB_NAME__" --replace_string="$db_name" --target_file="$aeneria_conf"
+# ynh_replace_string --match_string="__DB_USER__" --replace_string="$db_user" --target_file="$aeneria_conf"
+# ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd" --target_file="$aeneria_conf"
+# ynh_replace_string --match_string="__DESKEY__" --replace_string="$deskey" --target_file="$aeneria_conf"
```
Comment (SimonMellerin) [inline scripts/upgrade]: Are you sure keeping those comments is necessary ?

**Review by SimonMellerin — COMMENTED** [review]
thanks for your work on your time spend on this !

**Pull request merged**